dj/api/nodes.py

```diff
@@ -259,21 +270,55 @@ def delete_a_node(name: str, *, session: Session = Depends(get_session)):
         downstream.current.status = NodeStatus.INVALID
         session.add(downstream)
 
-    # If the node is a dimension, find all columns that
-    # are linked to this dimension and remove the link
-    if node.type == NodeType.DIMENSION:
-        columns = (
-            session.exec(select(Column).where(Column.dimension_id == node.id))
-            .unique()
-            .all()
+    now = datetime.utcnow()
+    node.deactivated_at = UTCDatetime(
+        year=now.year,
+        month=now.month,
+        day=now.day,
+        hour=now.hour,
+        minute=now.minute,
+        second=now.second,
+    )
+    session.add(node)
+    session.commit()
+    return JSONResponse(
+        status_code=HTTPStatus.NO_CONTENT,
+        content={"message": f"Node `{name}` has been successfully deactivated"},
+    )
+
+
+@router.post("/nodes/{name}/activate/", status_code=201)
+def activate_a_node(name: str, *, session: Session = Depends(get_session)):
+    """
+    Activate the specified node.
+    """
+    node = get_node_by_name(session, name, with_current=True, include_deactivated=True)
```

Comment:
nit: I think I see the following naming pattern:
- action: de/activate
- state: in/active
So perhaps this argument could be called `include_inactive` ?